src/include/processor/operator/profile.h

```diff
@@ -13,6 +13,17 @@ struct ProfileLocalState {
     bool hasExecuted = false;
 };
 
+struct ProfilePrintInfo final : OPPrintInfo {
+    ProfilePrintInfo() {}
+    ProfilePrintInfo(const ProfilePrintInfo& other) : OPPrintInfo{other} {}
+
+    std::string toString() const override { return "Explain"; }
```

Comment:
I think you don't need to print anything here. This should be captured in operator name. If so, remove this class.